concepts/runes/about.md

```diff
@@ -1,3 +1,120 @@
+
+## Unicode and Unicode Code Points
+
+Unicode is a superset of ASCII that represents characters by assigning a unique number to every character. This unique number is called a Unicode code point. Unicode aims to represent all the world's characters including various alphabets, numbers, symbols, and even emoji as Unicode code points.
+
+In Go, the `rune` type is meant to represent a single Unicode code point.
+
+The following table contains example Unicode characters along with their Unicode code point and decimal values:
+
+| Unicode Character | Unicode Code Point | Decimal Value |
+|-------------------|--------------------|---------------|
+| 0                 | `U+0030`           | `48`          |
+| A                 | `U+0041`           | `65`          |
+| a                 | `U+0061`           | `97`          |
+| ¿                 | `U+00BF`           | `191`         |
+| π                 | `U+03C0`           | `960`         |
+| 🧠                | `U+1F9E0`          | `129504`      |
+
+## UTF-8
+
+UTF-8 is a variable-width character encoding that is used to encode every Unicode code point as 1, 2, 3, or 4 bytes. Since a Unicode code point can be encoded as a maximum of 4 bytes, the `rune` type needs to be able to hold up to 4 bytes of data. That is why the `rune` type is an alias for `int32` as an `int32` type is capable of holding up to 4 bytes of data.
+
+Go source code files are encoded using UTF-8.
+
+## Using Runes
+
+Variables of type `rune` are declared by placing a character inside single quotes:
+
+```go
+myRune := '¿'
+```
+
+Since the underlying type of a rune is `int32`, printing a rune's type will yield `int32`:
+
+```go
+myRune := '¿'
+fmt.Printf("myRune type: %T\n", myRune)
+// Output: myRune type: int32
+```
+
+Similarly, printing a rune's value will yield the underlying integer (decimal) value:
+
+```go
+myRune := '¿'
+fmt.Printf("myRune value: %v\n", myRune)
+// Output: myRune value: 191
+```
+
+To print the Unicode character represented by the rune, use the `%c` formatting verb:
+
+```go
+myRune := '¿'
+fmt.Printf("myRune Unicode character: %c\n", myRune)
+// Output: myRune Unicode character: ¿
+```
+
+To print the Unicode code point represented by the rune, use the `%U` formatting verb:
+
+```go
+myRune := '¿'
+fmt.Printf("myRune Unicdoe code point: %U\n", myRune)
+// Output: myRune Unicode code point: U+00BF
+```
+
+## Runes and Strings
+
+Since Go source code files are encoded using UTF-8, strings in Go are also encoded using UTF-8. That means strings in Go contain Unicode characters. Since a `rune` type is meant to represent a Unicode character, a string in Go is often referred to as a sequence of runes. Taking this one step further, since runes are stored as 1, 2, 3, or 4 bytes, a string can also be referred to as a sequence of bytes. Go uses slices to represent such sequences, and these slices can be iterated over using `range`.
```

Comment:
From the text I find it hard to understand that a string is a slice of bytes but is NOT a slice of `rune`s because of the UTF8 encoding (but it can be converted to one as you explained elsewhere).

Reply:
Updated! Feel free to review.